### ISSUE TITLE
Add license

### ISSUE THREAD

**Issue by victoriacheng15** [music-os/awesome-music-web3-nfts#5]
I believe that you will need to add a license in order for others to make contributions to this repo.

**Comment by TimDaub** [comment]
awesome, let's try to relicense. @reimertz and @djfnd as co-contributors: Please comment if https://creativecommons.org/licenses/by-nc-sa/4.0/ is a license you agree with here.

**Comment by djfnd** [comment]
Yeah agree, some kind of CC licence will fit well. Any specific reason for including the conditions though? This list is intended to be a compilation of publicly available resources, right? So why don't we just go full CC0?

**Comment by TimDaub** [comment]
I'm also OK with CC0. @reimertz pls comment ur preference. You're a co-contributor so without you we can't relicense.
